5. Linting integration README (#5666)
DataDog/integrations-core: mapr/README.md

## Troubleshooting

- **The Agent is on a crash loop after configuring the MapR integration**

>   There have been a few cases where the C library within _mapr-streams-python_ segfaults, and they were explained by permissions issues. Make sure the `dd-agent` user has read permission on the ticket file, that the `dd-agent` user is able to run maprcli commands when the MAPR_TICKETFILE_LOCATION environment variable points to the ticket.

Reviewer: ```suggestion
  There have been a few cases where the C library within _mapr-streams-python_ segfaults because of permissions issues. Make sure the `dd-agent` user has read permission on the ticket file, that the `dd-agent` user is able to run maprcli commands when the MAPR_TICKETFILE_LOCATION environment variable points to the ticket.
```
